xgi/readwrite/xgi_data.py

```diff
@@ -1,20 +1,28 @@
 import requests
+import json
+import os
 
 from .. import convert
 from ..exception import XGIError
 
-__all__ = ["load_xgi_data"]
+__all__ = ["load_xgi_data", "download_xgi_data", "_clean_file_path", 
```

Comment:
Do we want/need to make the private functions _clean_file_path and _request_from_xgi_data available? If they might be useful somewhere else we could make them non-private or even put them in utilities.py

Reply:
I don't think that they are going to be useful anywhere else. I am used to making all functions available by default (because why not). But we can make them unavailable if you prefer it that way?

Reply:
I saw you made them unavailable now, let's leave it like this and change later if we need it.